src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -15,6 +15,10 @@
                                 @update="updateSelectedFilterOptions(ds.config.id, $event)">
                     </filters>
                 </div>
+                <div id="chart-description"
+                     v-if="chartConfigValues.description"
```

Comment:
I don't think you need this if statement, description value is static and will always be true.

Reply:
It comes from the chart configuration metadata - in the case of the Input Time Series it will be true as currently configured, but the type is defined to have that value optional (see [here](https://github.com/mrc-ide/hint/pull/631/files#diff-b4e97f9053896e4bab011155c7ed2ad40cf453d0da07ccbb80ae5144a9424c23R238)) so it could be undefined.